src/pages/domains.mdx

```diff
@@ -10,7 +10,8 @@ import DomainList from '../components/DomainList.astro';
 
 # Erisa's domain list
 
-This list is designed to be machine readable.  
-You can find the raw text file [here](/domains.txt).
+This list is designed to be machine readable.
+
+You can find the raw text file [here](/domains.txt) if it doesn't load for you. Sorry for the trouble!
```

Comment:
```suggestion
This list is designed to be machine readable.  
You can find the raw text file [here](/domains.txt).
```
I liked how this was before. no need for the "if this doesnt load" because its part of the html and will always load